src/Locales/extract_messages.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+which xgettext &>/dev/null || { echo "xgettext not found, please install it"; exit 1; }
+
+cd "$( dirname "$0" )/../"
```

Comment:
I would suggest adding `--` to `dirname` and `cd` to support file names starting with "-" (rarely seen indeed):

```Shell
cd -- "$( dirname -- "$0" )/../"
```

Reply:
> I would suggest adding `--` to `dirname` and `cd` to support file names starting with "-" (rarely seen indeed):
> 
> ```shell
> cd -- "$( dirname -- "$0" )/../"
> ```
Good idea: done ! Thanks.